GameDataParser/Parsers/ItemParser.cs

```diff
@@ -53,30 +66,37 @@ public static class ItemParser {
             return items;
         }
 
-        public static void Write(List<ItemMetadata> items) {
-            using (FileStream writeStream = File.OpenWrite(OUTPUT)) {
+        public static void Write(List<ItemMetadata> items)
+        {
+            using (FileStream writeStream = File.OpenWrite(VariableDefines.OUTPUT + "ms2-item-metadata"))
+            {
                 Serializer.Serialize(writeStream, items);
             }
-            using (FileStream readStream = File.OpenRead(OUTPUT)) {
+            using (FileStream readStream = File.OpenRead(VariableDefines.OUTPUT + "ms2-item-metadata"))
```

Comment:
Is this `using` block doing anything?

Reply:
Serializing the file to export, and that is just use there, instead of calling at beging

Reply:
It looks like it's just reading the file but doing nothing with it

Reply:
To serialize an object it needs a buffer and the data, in this case is using the file that will be export for buffer and then pasing the data.